doc/metrics.md

```diff
@@ -293,10 +293,10 @@ can be used to information about the record measurement itself.
 
 ```typescript
 async function myTask() {
-  const httpServerDuration = meter.createHistogram("http.server.duration", {
-    description: 'A http server duration',
-    unit: 'milliseconds',
-    valueType: ValueType.INT
+  const httpServerDuration = meter.createHistogram("my.http.server.request.duration", {
```

Comment:
Based on the other docs examples I'm wondering if this was intentionally supposed to match the semantic convention attribute name... in which case it should be imported in I think as `METRIC_HTTP_SERVER_REQUEST_DURATION` right?

```suggestion
  const httpServerDuration = meter.createHistogram(METRIC_HTTP_SERVER_REQUEST_DURATION, {
```